docs/serving/getting-started-knative-app.md

```diff
@@ -36,24 +35,41 @@ docker tag local-image dev.local/local-image
 
 [Learn more about image tag resolution.](./tag-resolution.md)
 
-The Hello World sample app reads in an `env` variable, `TARGET`, from the
-configuration `.yaml` file, then prints "Hello World: \${TARGET}!". If `TARGET`
-isn't defined, it will print "NOT SPECIFIED".
+The Hello World sample app reads in an `env` variable, `TARGET`, then prints "Hello World: \${TARGET}!". If `TARGET` isn't defined, it will print "NOT SPECIFIED".
 
-## Configuring your deployment
+## Creating your Deployment with the Knative CLI
 
-To deploy an app using Knative, you need a configuration `.yaml` file that
-defines a Service. For more information about the Service object, see the
+The easiest way to create Knative service deployment is by using the Knative CLI [kn](https://github.com/knative/client).
+You can install the `kn` binary as described in [Installing the Knative CLI](../install/install-kn.md)
+
+`kn` allows you to create a service interactively on the command line.
+Eventually, it will create a corresponding resource description internally as when using a YAML file directly.
```

Comment:
I don't understand what this sentence ("Eventually, ...") means. Can we remove it?